### DISCUSSION TITLE
403 error on bestbuy

### DISCUSSION THREAD

**Discussion by ghost** [jef/streetmerchant#1347] [Help]
I'm running streetmerchant on google cloud vm. Bestbuy gives me a 403 error right after I start the bot. Are there any fixes to this. I have tor and I have set a proxy through MaxCircuitDirtiness but that doesn't seem to do anything. Do any of you have a potential fix to this?

**Comment by ScottEgan** [comment]
Not entirely sure but I think Best Buy has some sort of rate limit. Maybe try checking less often? I found between 90s to 120s helped eliminate the 403 error.

**Comment by holeyness** [comment]
I dealt with this problem by sending out the request sequentially instead of all together for each of the stores, i.e. await the previous iteration of the loop to finish before sending out the next
